PancakeSwap-Bots/Auto-CAKE-Admin-Operations/src/agent.ts

```diff
@@ -0,0 +1,34 @@
+import { HandleTransaction, TransactionEvent, getEthersProvider, ethers, Initialize, Finding } from "forta-agent";
+import { NetworkManager } from "forta-agent-tools";
+import eventsBot from "./events.listener";
+import functionCallBot from "./function.call.listener";
+import { NetworkData, DATA } from "./config";
+
+const networkManager = new NetworkManager(DATA);
+
+const provideInitialize = (
+  networkManager: NetworkManager<NetworkData>,
+  provider: ethers.providers.Provider
+): Initialize => {
+  return async () => {
+    await networkManager.init(provider);
+  };
+};
+const provideHandleTransaction = (networkManager: NetworkManager<NetworkData>): HandleTransaction => {
```

Comment:
```suggestion
export const provideHandleTransaction = (networkManager: NetworkManager<NetworkData>): HandleTransaction => {
```
We typically add `export` to the declaration instead of adding it to the object declared as `export default`.